src/ucp/tag/tag_recv.c

```diff
@@ -160,11 +168,13 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_tag_recv_nbr,
                  void *request)
 {
     ucp_request_param_t param = {
-        .op_attr_mask = UCP_OP_ATTR_FIELD_DATATYPE |
-                        UCP_OP_ATTR_FIELD_REQUEST  |
+        .op_attr_mask = UCP_OP_ATTR_FIELD_DATATYPE    |
+                        UCP_OP_ATTR_FIELD_REQUEST     |
+                        UCP_OP_ATTR_FIELD_MEMORY_TYPE |
```

Comment:
no need to add UCP_OP_ATTR_FIELD_MEMORY_TYPE param, the default is already UNKNOWN